src/js/chapter-data.js

```diff
@@ -4427,7 +4427,7 @@ export const chapter7Data = [{
 
     Egestion is the removal of faecal matter (poop) from the body.
 
-    ![Image](https://s3-ap-southeast-1.amazonaws.com/chalees-min-images/learning-items/grade7/ingestion-digestion-egestion-gif3.gif)
+    ![Image](https://media.giphy.com/media/3oKIPjWupbEMgF0FZ6/giphy.gif)
```

Comment:
I see a CORS error while we try to fetch the image from the AWS bucket. You might want to configure CORS on AWS. react-gif-player might not be able fetch still images from the gif because of that. 
While hosting on giphy i don't get the CORS error and everything works fine.